src/shared/components/palette/dynamic-radio-list/read-dynamic-radio-list-field.component.spec.ts

```diff
@@ -0,0 +1,136 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { ReadDynamicRadioListFieldComponent } from './read-dynamic-radio-list-field.component';
+import { DebugElement } from '@angular/core';
+import { FieldType } from '../../../domain/definition/field-type.model';
+import { DynamicRadioListPipe } from './dynamic-radio-list.pipe';
+import { CaseField } from '../../../domain/definition/case-field.model';
+import { FormGroup } from '@angular/forms';
+
+describe('ReadDynamicRadioListFieldComponent', () => {
+
+  const VALUE = 'F';
+  const EXPECTED_LABEL = 'Female';
+  const FIELD_ID = 'ReadOnlyFieldId';
+  const FIELD_TYPE: FieldType = {
+    id: 'Gender',
+    type: 'DynamicRadioList',
+    fixed_list_items: [
+      {
+        code: 'M',
+        label: 'Male',
+        order: 1
+      },
+      {
+        code: VALUE,
+        label: EXPECTED_LABEL,
+        order: 2
+      },
+      {
+        code: 'O',
+        label: 'Other',
+        order: 3
+      }
+    ]
+  };
+
+  describe('Non-persistable readonly fixed radio list field', () => {
+    const CASE_FIELD: CaseField = <CaseField>({
+      id: 'x',
+      label: 'X',
+      display_context: 'OPTIONAL',
+      field_type: FIELD_TYPE,
+      value: VALUE
+    });
+    const EMPTY = '';
+
+    let fixture: ComponentFixture<ReadDynamicRadioListFieldComponent>;
+    let component: ReadDynamicRadioListFieldComponent;
+    let de: DebugElement;
+
+    beforeEach(async(() => {
+      TestBed
+        .configureTestingModule({
+          imports: [],
+          declarations: [
+            ReadDynamicRadioListFieldComponent,
+            DynamicRadioListPipe
+          ],
+          providers: []
+        })
+        .compileComponents();
+
+      fixture = TestBed.createComponent(ReadDynamicRadioListFieldComponent);
+      component = fixture.componentInstance;
+
+      component.caseField = CASE_FIELD;
+
+      de = fixture.debugElement;
+      fixture.detectChanges();
+    }));
+
+    it('render label associated to the value provided', () => {
```

Comment:
Being finicky and for readability I would say 'should render' for this and the other tests that don't have it